gui/workorder-finetune

```diff
@@ -0,0 +1,544 @@
+-- adjust work orders' input item, material, traits
+--[====[
+
+gui/workorder-finetune
+======================
+Adjust input items, material, or traits for work orders. Actual
+jobs created for it will inherit the details.
+
+This is the equivalent of `gui/workshop-job` for work orders,
+with the additional possibility to set input items' traits.
+
+It has to be run from a work order's detail screen
+(:kbd:`j-m`, select work order, :kbd:`d`).
+
+For best experience add the following to your ``dfhack*.init``::
+
+    keybinding add D@workquota_details gui/workorder-finetune
+
+]====]
+
+--[[
+credit goes to the author of `gui/workshop-job` where the majority
+of the code here comes from.
```

Comment:
now that you're reusing the `gui/workshop-job` code directly, is this still true?

Reply:
changed the wording a bit there